packages/ses/src/enablements.js

```diff
@@ -85,6 +86,7 @@ export const moderateEnablements = {
     // may eventually be fixed at the source.
     // hasOwnProperty: true, // set by "vega-util".
 
+    toLocaleString: true, // set by https://github.com/feross/buffer
```

Comment:
@kumavis if your use of feross/buffer works with this line, please try commenting it out and see if your use of feross/buffer still works.

With just this line commented out, my own test cases here will fail. But I'll just need to revert the changes in this PR for testing this case.